### PR TITLE
Return blinding pubkey with blinding nonces

### PR DESCRIPTION
**Pull request by jgriffiths**

### CODE REVIEW

File: green_cli/authenticators/__init__.py [hunk 1]
```diff
@@ -67,7 +67,7 @@ def __init__(self, options):
 
     @property
     def _mnemonic(self):
-        return self.mnemonic_prop.get()
+        return self.mnemonic_prop.get().strip()
```
Comment (jkauffman1) [inline green_cli/authenticators/__init__.py:70]: I think this is trying to do the same thing as `SoftwareAuthenticator.set_mnemonic` which does a split/join, I wonder if there should be a `MnemonicOnDisk.normalise` or some such that is called by the getter and the setter.

**Comment by jkauffman1** [comment]
utack fbba957a889a29fa3254f47abbb9dbf39d495cb5

File: green_cli/authenticators/__init__.py (hunk 1 above)
Comment (jgriffiths) [reply green_cli/authenticators/__init__.py]: Updated to normalize all the things

**Comment by jkauffman1** [comment]
utack d27c6688ac83cfc82843d9cb7fd4fa3149334fa9

**Pull request merged**